resources/views/projects/my-projects.blade.php

```diff
@@ -18,54 +18,73 @@
         ($user->context === 'individual' &&
             ($user->individual->isConsultant() || $user->individual->isConnector()) &&
             $user->individual->isParticipant()))
-        <nav class="nav--tabbed" aria-labelledby="projects">
-            <div class="center center:wide">
-                <ul class="-mt-4 flex gap-6" role="list">
-                    @if ($user->context === 'organization')
-                        @if ($user->organization->isConsultant() || $user->organization->isConnector())
+        @if ($user->context === 'organization' ||
+            ($user->context === 'individual' && $user->individual->roles && sizeof($user->individual?->roles ?? []) > 1) ||
```

Comment:
`$user->individual->roles && sizeof($user->individual?->roles ?? []) > 1` is a bit redundant. You can just check 

```suggestion
            ($user->context === 'individual' && count($user->individual?->roles ?? []) > 1) ||
```